src/webswview/lkswview/api/views.py

```diff
@@ -16,4 +25,32 @@ def get(self, id):
         return {'task': 'Say "Hello, World!"'}
 
 
+class SrcPackage(Resource):
+    def get(self, repo_name, suite_name, name):
+        with session_scope() as session:
+            rss = repo_suite_settings_for(session, repo_name, suite_name, fail_if_missing=False)
+            if not rss:
+                abort(404)
+
+            spkgs = (
+                session.query(SourcePackage)
+                .options(undefer(SourcePackage.version))
+                .filter(SourcePackage.repo_id == rss.repo_id)
+                .filter(SourcePackage.suites.any(ArchiveSuite.id == rss.suite_id))
+                .filter(SourcePackage.name == name)
+                .filter(SourcePackage.time_deleted.is_(None))
+                .order_by(SourcePackage.version.desc())
+                .all()
+            )
+            if not spkgs:
+                abort(404)
+
+            return {
+                'pkgs': spkgs,
```

Comment:
This will not work, a `SourcePackage` entity will not magically convert itself into JSON (sadly). So you'd need to create a method that converts `SourcePackage` into dict form.